Riot/Modules/Common/Recents/DataSources/RecentsDataSource.m

```diff
@@ -1167,13 +1172,14 @@ - (NSIndexPath*)cellIndexPathWithRoomId:(NSString*)roomId andMatrixSession:(MXSe
             {
                 return nil;
             }
-            indexPath = [NSIndexPath indexPathForRow:index inSection:serverNoticeSection];
+            NSInteger sectionIndex = [self.sections sectionIndexForSectionType:RecentsDataSourceSectionTypeServerNotice];
+            indexPath = [NSIndexPath indexPathForRow:index inSection:sectionIndex];
         }
     }
 
-    if (!indexPath && (suggestedRoomsSection >= 0))
+    if (!indexPath && ([self.sections contains:RecentsDataSourceSectionTypeSuggestedRooms]))
     {
-        index = [self cellIndexPosWithRoomId:roomId andMatrixSession:matrixSession within:self.serverNoticeCellDataArray];
+        index = [self cellIndexPosWithRoomId:roomId andMatrixSession:matrixSession within:self.suggestedRoomCellDataArray];
```

Comment:
This was previously set to `serverNoticeCellDataArray`, which I think is an error / bug

Reply:
Yes it's, good catch 👍